src/bin/common.rs

```diff
@@ -705,5 +712,8 @@ fn parse_config(matches: &ArgMatches<'_>) -> Result<EncoderConfig, CliError> {
 
   cfg.low_latency = matches.is_present("LOW_LATENCY");
 
+  // Disables scene-detection
+  cfg.no_scene_detection = matches.is_present("NO_SCENE_DETECTION");
```

Comment:
It better to do it in this way
If scene detection would be disabled for speed, it will be respected in future:
```rust 
  if matches.is_present("NO_SCENE_DETECTION") {
    cfg.no_scene_detection = true;
  } else {
    cfg.no_scene_detection = SpeedSettings::no_scene_detection(speed);
  }
```